examples/src/main/java/com/mapbox/navigation/examples/ui/NavigationViewActivity.kt

```diff
@@ -106,21 +108,29 @@ class NavigationViewActivity : AppCompatActivity(), OnNavigationReadyCallback, N
     }
 
     override fun onNavigationRunning() {
-    // todo
+        // todo
     }
 
     override fun onNavigationFinished() {
-    // todo
+        // todo
     }
 
     override fun onCancelNavigation() {
         navigationView.stopNavigation()
         finish()
     }
 
+    private fun getInitialCameraPosition(): CameraPosition {
+        return CameraPosition.Builder()
+            .target(LatLng(37.791503, -122.396255))
```

Comment:
You can get the `origin` from the `DirectionsRoute` after parsing it in `getDirectionsRoute()`